src/services/dex.js

```diff
@@ -1078,41 +1078,42 @@ export default {
             }
 
             const history = {
+              apiBuckets: await this.fetchTradesBucketed(marketName),
               date: res.data.timestamp,
+              getBars: this.getTradeHistoryBars,
               marketName,
               trades: res.data.trades,
-              getBars: this.getTradeHistoryBars,
             };
             const todayCutoff = moment().startOf('day').unix();
             const todayTrades = _.filter(history.trades, (trade) => {
```

Comment:
This only includes the last 100 trades that occurred. It won't have all the trades from the last 24 hours, so the only thing that will be accurate from this is the close24Hour that is calculated.